### PR TITLE
Feature/mayuri

### PR DESCRIPTION
**Pull request by mayurigurav1997**

### CODE REVIEW

**Comment by MayuriPatil1912** [comment]
Mobile

Desktop

File: components/dropdown/dropdown.tsx [hunk 1]
```diff
@@ -4,43 +4,52 @@ import {
...
+  FormHelperText,
+  InputLabel
 } from "@mui/material";
 import React, { useState } from "react";
 
 interface DropdownProps {
   // Define your expected properties here
   options: Array<{ value: string; label: string }>;
   label: string;
+  register:any;
```
Comment (nileshv1) [inline components/dropdown/dropdown.tsx:16]: Avoid using "any" as type, give valid type to variable.

File: components/dropdown/dropdown.tsx [hunk 2]
```diff
@@ -4,43 +4,52 @@ import {
...
-    setDropdownValue(event.target.value);
-  };
-  console.log(dropdownValue);
+  const { options, label, register, required, errors, ...rest } = props;
+  // console.log(props)
+  // const [dropdownValue, setDropdownValue] = useState("");
+  // const handleChange = (event: SelectChangeEvent) => {
+  //   setDropdownValue(event.target.value);
+  // };
+  // console.log(dropdownValue);
```
Comment (nileshv1) [inline components/dropdown/dropdown.tsx]: remove commented code, console.log

File: components/dropdown/dropdown.tsx [hunk 3]
```diff
@@ -4,43 +4,52 @@ import {
...
-        renderValue={(selected) => {
-          console.log(selected, "selected");
-          if (selected.length === 0) {
-            return <em>Placeholder</em>;
-          }
-          return selected;
-        }}
+        // label={label}
+        {...register(label, {required})}
+        // renderValue={(selected) => {
```
Comment (nileshv1) [inline components/dropdown/dropdown.tsx]: remove commented code

**Review by nileshv1 — APPROVED** [review]

**Pull request merged**